test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -74,6 +74,8 @@
     ('aten::as_tensor', datetime.date(2020, 6, 30)),
 ]
 
+white_list.append(("aten::list_with_default", datetime.date(2020, 6, 30)))
```

Comment:
why is this an append rather than listed in the list explicitly?

Reply:
it won't merge conflict, which has happened to me in the past with the bc list quite a lot